### PR TITLE
Bug 1525719: create a non-production /ducks/ endpoint for the React spike

### PR DESCRIPTION
**Pull request by davidflanagan**
This patch adds a new /ducks/ url just like our existing /docs/ url
for wiki documents, except it has a duck in the header. The URL route
is not installed if we're in production so this should be safe. We
are going to use this new url for our experimentation with React.

### CODE REVIEW

**Comment by davidflanagan** [comment]
screen shot 2019-02-08 at 12 24 01 pm

**Comment by davidflanagan** [comment]
The react_*.html templates files are copies of existing templates, barely changed, except for the addition of ducks.

The react_document() method in document.py is a copy of document(), except with a different template file.

The interesting changes to be reviewed are in kuma/urls.py and kuma/wiki/urls.py

**Comment by davidflanagan** [comment]
I improved the logo!

screen shot 2019-02-08 at 1 19 40 pm

File: kuma/urls.py [hunk 1]
```diff
@@ -89,6 +90,12 @@ def serve_from_media_root(request, path):
 urlpatterns += i18n_patterns(url(r'^docs.json$', document_as_json,
                                  name='wiki.json'))
 urlpatterns += i18n_patterns(url(r'^docs/', include(wiki_lang_urlpatterns)))
+
+# A special endpoint for our React spike, local and staging only
+if settings.DOMAIN != settings.PRODUCTION_DOMAIN:
```
Comment (jwhitlock) [inline kuma/urls.py]: I double-checked, this will work correctly - it will show up in staging and development, but will be a 404 in production.

**Comment by jwhitlock** [comment]
@davidflanagan you may want to remove the CodePen stuff that was removed in https://github.com/mozilla/kuma/pull/5243

**Comment by davidflanagan** [comment]
Thanks for catching that, John. Done.

**Pull request merged**